draft-ietf-quic-multipath.md

```diff
@@ -976,7 +949,7 @@ PATH_STATUS frames are formatted as shown in {{fig-path-status-format}}.
 ~~~
   PATH_STATUS Frame {
     Type (i) = TBD-03 (experiments use 0xbaba06),
-    Path Identifier (..),
+    Packet Number Space Identifier (i),
```

Comment:
_Path Identifier (i),_

Reply:
That was an oversight. Fixed now! Thanks!